discord/app/context.py

```diff
@@ -61,6 +61,10 @@ def message(self):
     @cached_property
     def user(self):
         return self.interaction.user
+    
+    @cached_property
+    def author(self):
+        return self.interaction.user
```

Comment:
Doesn't look like this is used. Plus it's the exact same as `user`.

Plus I'd argue that `author` doesn't make a ton of sense in the context of a slash command, as there's no message to author.

Reply:
Yeah you can ignore this, it was from a previous PR. I didnt know it would show up here.
But at the time it was just to help it feel familiar to those just getting into slash commands. Most people already know author as the command user. 

I believe this was already implemented though